docs/src/userguide/numpy_tutorial.rst

```diff
@@ -462,7 +355,9 @@ compile-time if the type is set to :obj:`np.ndarray`, specifically it is
 assumed that the data is stored in pure strided mode and not in indirect
 mode).
 
-[:enhancements/buffer:More information]
+Where to go from here?
+======================
```

Comment:
One more thing to hint at here is the Pythran support. The drawback is that it really only currently works with the buffer syntax and not yet with memory views. So now, after changing all of this tutorial to memoryviews (which is great!), it doesn't really line up anymore.